syft/pkg/cataloger/javascript/parse_package_json.go

```diff
@@ -41,14 +43,16 @@ type author struct {
 	URL   string `json:"url" mapstructure:"url"`
 }
 
+type authors []author
+
 type repository struct {
 	Type string `json:"type" mapstructure:"type"`
 	URL  string `json:"url" mapstructure:"url"`
 }
 
 // match example: "author": "Isaac Z. Schlueter <i@izs.me> (http://blog.izs.me)"
 // ---> name: "Isaac Z. Schlueter" email: "i@izs.me" url: "http://blog.izs.me"
-var authorPattern = regexp.MustCompile(`^\s*(?P<name>[^<(]*)(\s+<(?P<email>.*)>)?(\s\((?P<url>.*)\))?\s*$`)
+var authorPattern = regexp.MustCompile(`^\s*(?P<n>[^<(]*)(\s+<(?P<email>.*)>)?(\s\((?P<url>.*)\))?\s*$`)
```

Comment:
I don't think this needs to change from a more descriptive `name`, does it?

Reply:
No, that was my bad. I'll put that back, thanks.